Makefile

```diff
@@ -67,6 +67,18 @@ golden-diff: commodore_args += -f tests/$(instance).yml
 golden-diff: clean .compile ## Diff compile output against the reference version. Review output and run `make gen-golden golden-diff` if this target fails.
 	@git diff --exit-code --minimal --no-index -- tests/golden/$(instance) compiled/
 
+.PHONY: golden-diff-all
```

Comment:
The new make targets etc. will be removed by modulesync, you need to configure the test matrix in `.sync.yml`, see e.g. https://github.com/projectsyn/component-cilium/blob/aec1e6a4d5886eeadfc8f03e5eec9242eb40d303/.sync.yml#L5-L14. Also you probably want to run the sync for this component afterwards, so you also get the new GitHub actions config.

Reply:
yeah, there's an action you can trigger manually in the modulesync repo